.github/ISSUE_TEMPLATE/documentation-update.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: Documentation request
+about: Use this to propose information improvements and additions
+title: "[ Documentation request ] "
+labels: ''
+assignees: ''
```

Comment:
We can remove this field since there's no assignee

Reply:
Done